catalog/task/slack-webhook-notification/0.1/slack-webhook-notification.yaml

```diff
@@ -0,0 +1,68 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: "release"
+  name: slack-webhook-notification
+spec:
+  description: >-
+    Sends message to slack using incoming webhook
+  params:
+    - name: message
+      description: Message to be sent
+    - name: extraDataJsonPath
+      description: path to extraData json file
+      default: "$(workspaces.data.path)/extra_data.json"
+    - name: secretName
+      description: |
```

Comment:
same comment as in README

Reply:
Updating it to the same in the README now would suffice

Reply:
fixed